config/initializers/commontator.rb

```diff
@@ -1,187 +1,249 @@
+  # Note: can be called with a user object that is nil (if they are not logged in)
+  # Default: lambda { |thread, user| true } (anyone can read any thread)
+  config.thread_read_proc = lambda { |thread, user| true }
+
+  # thread_moderator_proc
+  # Type: Proc
+  # Arguments: a thread (Commontator::Thread), a user (acts_as_commontator)
+  # Returns: a Boolean, true iif the user is a moderator for that thread
+  # If you want global moderators, make this proc true for them regardless of thread
+  # Default: lambda { |thread, user| false } (no moderators)
+  config.thread_moderator_proc = lambda { |thread, user| false }
+
+  # comment_editing
+  # Type: Symbol
   # Whether users can edit their own comments
-  # after someone posted a newer comment
-  # Default: false
-  config.can_edit_old_comments = false
-
-  # Whether users can delete their own comments
-  # Default: true
-  config.can_delete_own_comments = false # REVIEW: Temporarily disabled
-
+  # Valid options:
+  #   :a (always)
+  #   :l (only if it's the latest comment)
+  #   :n (never)
+  # Default: :l
+  config.comment_editing = :l
+
+  # comment_deletion
+  # Type: Symbol
   # Whether users can delete their own comments
-  # after someone posted a newer comment
-  # Default: false
-  config.can_delete_old_comments = false # REVIEW: Temporarily disabled
-
-  # Whether users can manually subscribe or unsubscribe to threads
-  # Default: true
-  config.can_subscribe_to_thread = false # REVIEW: Temporarily disabled
-
+  # Valid options:
+  #   :a (always)
+  #   :l (only if it's the latest comment)
+  #   :n (never)
+  # Note: For moderators, see the next option
+  # Default: :l
+  config.comment_deletion = :l
+
+  # moderator_permissions
+  # Type: Symbol
+  # What permissions moderators have
+  # Valid options:
+  #   :e (delete and edit comments and close threads)
+  #   :d (delete comments and close threads)
+  #   :c (close threads only)
+  # Default: :d
+  config.moderator_permissions = :d
+
+  # comment_voting
+  # Type: Symbol
   # Whether users can vote on other users' comments
-  # Note: requires acts_as_votable gem installed
-  # and configured for your application
-  # Default: false
-  config.can_vote_on_comments = false # REVIEW: Temporarily disabled
-
+  # Valid options:
+  #   :n  (no voting)
+  #   :l  (likes - requires acts_as_votable gem)
+  #   :ld (likes/dislikes - requires acts_as_votable gem)
+  # Not yet implemented:
+  #   :s  (star ratings)
+  #   :r  (reputation system)
+  # Default: :n
+  config.comment_voting = :n
+
+  # vote_count_proc
+  # Type: Proc
+  # Arguments: a thread (Commontator::Thread), pos (Fixnum), neg (Fixnum)
+  # Returns: vote count to be displayed (String)
+  # pos is the number of likes, or the rating, or the reputation
+  # neg is the number of dislikes, if applicable, or 0 otherwise
+  # Default: lambda { |thread, pos, neg| "%+d" % (pos - neg) }
+  config.vote_count_proc = lambda { |thread, pos, neg| "%+d" % (pos - neg) }
```

Comment:
Unused block argument - `thread`. If it's necessary, use `_` or `_thread` as an argument name to indicate that it won't be used.